src/upgrade/upgrade_scripts/5.14.0/upgrade_bucket_policy.js

```diff
@@ -0,0 +1,53 @@
+/* Copyright (C) 2023 NooBaa */
+"use strict";
+
+const util = require('util');
+const _ = require('lodash');
+const { OP_NAME_TO_ACTION } = require('../../../endpoint/s3/s3_utils');
+
+async function run({ dbg, system_store, system_server }) {
+
+    try {
+        dbg.log0('Starting bucket policy upgrade...');
+        const buckets = [];
+        const all_op_names = new Map(_.flatMap(OP_NAME_TO_ACTION, action =>
```

Comment:
Building the Map/Object using a for loop will iterate OP_NAME_TO_ACTION only once instead of 3 times currently